### ISSUE TITLE
Replace Touch controllers with hand models?

### ISSUE THREAD

**Issue by vr-fun** [Eusth/HoneySelectVR#47]
Any possibility to replace the Touch controller models with hands? I believe this is part of various SDKs, and could also allow hand gestures (pointing fingers, open/closed hands, etc.). No other change to UX would be necessary.

This would increase immersion tremendously. I don't know if this is a feature request for the HSVR mod or the underlying VRgin code.

Otherwise, the .6 version is working excellently.

**Comment by Eusth** [comment]
This is definitely something I'd like to implement.

The current version actually already comes with hand models that were included for the Leap Motion support. In theory, I could just use them for a representation. The hands that come with the Oculus SDK would also be cool, but since I'm using OpenVR that's out of reach.

I'll look into it when I have some time on my ... hands.

**Comment by deinlandel** [comment]
It would be great even for Vive Wands. Seeing hands is definitely more immersive than seeing controllers.

**Comment by Eusth** [comment]
By the way, I actually built a feature into Play Club VR that lets you control the hands of the person you're impersonating. Was just a quick hack, though.

**Comment by deinlandel** [comment]
Yeah, interaction in Play Club is cool. I hope there will be maestro mode in Honey Select one day. =)
